### ISSUE TITLE
3 < 10 is false? Value::Addr < Value::Integer PartialOrd mismatching

### ISSUE THREAD

**Issue by phorward** [tokay-lang/tokay#21]
It seems there is a problem with a Value::Addr() and Value::Integer, as shown in the following example.
```
$ tokay
Tokay 0.4.0
>>> l = (1,2,3)
>>> l.len
3
>>> a = l.len
>>> a
3
>>> a < 10
false
>>> a > 10
true
```
This works:
```
>>> b = 3
>>> b < 10
true
>>> b > 10
false
```

**Comment by phorward** [comment]
Reproducing playground https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=74a0441492b4e2ce954ca1eb6ca8412e

**Comment by phorward** [comment]
Will be resolved by #9
